index.css

```diff
@@ -369,3 +393,120 @@ form {
   display: flex;
   flex-grow: 1;
 }
+
+/* Large screen devices */
+@media screen and (min-width: 600px) {
```

Comment:
Thank you for using media queries for the desktop breakpoint :+1: 

However, your selected `min-width: 600px` is too small as a breakpoint for desktop/large devices. Typically, you should use something around `992px` as a minimum breakpoint for large devices. This is so because we cannot categorize devices of width 600px as large devices. 

I understand that there are different resources with different opinions about what the exact breakpoint should be,  `992px` is widely used as a minimum breakpoint for large devices. Therefore, I think you should stick to that. 

If you're interested, you can read more [here](https://www.w3schools.com/css/css_rwd_mediaqueries.asp)